templates/components/_header.html

```diff
@@ -26,7 +26,7 @@
 								<li class="menu-item-has-children">
 									<a href="javascript:void(0);">More</a>
 									<ul class="sub-menu">
-										<li><a href="/#speakers">Speakers</a></li>
+										<li><a href="#speakers">Speakers</a></li>
```

Comment:
Agreed.. But using /#tagname will cause it to link to root of website and incase the website is running at say example.xyz/subdirectory/ and not example.xyz, it will link to example.xyz/#tagname instead of example.xyz/subdirectory/#tagname